src/index.ts

```diff
@@ -43,6 +43,7 @@ import BottomSheet, { BottomSheetProps } from './bottomSheet/BottomSheet';
 import LinearProgress, { LinearProgressProps } from './linearProgress';
 import Switch, { SwitchProps } from './switch/switch';
 import Tab, { TabItemProps, TabProps } from './tab/tab';
+import FloatingInput, { FloatingInputProps } from './input/floatingInput';
```

Comment:
Wouldn't it be better if add `variant='floating'` or `variant='outlined'` Prop in input? What do you think?

Reply:
I wanted first to get the core styling of the component done and then build these new props in future iterations. that being said, that prop is indeed necessary so I'll look into it for sure.

Reply:
^ I agree with Arpit here. This seems like more of a variant rather than a completely separate new component.